packages/server/postgres/queries/OrgUserAudit.ts

```diff
@@ -0,0 +1,57 @@
+/** Types generated for queries found in "packages/server/postgres/queries/OrgUserAudit.sql" */
```

Comment:
can we keep generated files in their own directory? it's always nice to be able to easily blow away a generated directory when something goes wrong or exclude from git if we need to down the road

Reply:
yeah sure!